### ISSUE TITLE
Queue script not running on Blitz

### ISSUE THREAD

**Issue by quatick** [quirrel-dev/quirrel#215]
## Bug Report
I am trying to run the example (adapted) from the documentation [https://docs.quirrel.dev/recipes/processing-upload-in-background](url) in **local development** but the code inside the `Queue` function doesn't run.

**Current Behavior**
The `UploadedCSV` record is created with success and call the `csvProcessingQueue.enqueue` is listed in https://ui.quirrel.dev/activity with the statuses scheduled/started/completed as expected, but the code inside (which I reduce to the minimum (only the `await db.uplodadedCsv.delete({where: {id: uploadId}})` part) is not running.

Also, I don't know if it is also an expected behavior but no console.log are showing on the console either.

**Input Code**

_Here is the most reduce sample it could provide :_ 

`in app/api/upload-image.ts`
```ts
import imageQueue from "app/api/upload-queue"

const uploadImage = async (req, res) => {
    // The poiImageToUpload id 13 exist in the database
    await imageQueue.enqueue(13)

    res.status(201).send({ data: "Successfully created" })
}

export default uploadImage
```

`in app/api/image-queue.ts`
```ts
import { Queue } from "quirrel/blitz"
import db from "db"

export default Queue(
  "api/image-queue",
  async (id: number) => {    
    await db.poiImageToUpload.delete({ where: { id } })
  }
)
```

**Environment**

- Quirrel version: 1.3.4
- Node/npm version: 14.16.0/6.14.11
- Blitz version 0.33.0 (and also tried 0.32.1)

**Comment by Skn0tt** [comment]
Hi @quatrick! Thanks for the very thorough bug report. I can't immediately spot an error from it. Could you provide a reproduction or give me temporary access to the code? Feel free to reach out via support@quirrel.dev if it's sensitive.

**Comment by Skn0tt** [comment]
@quatick gave me access to the code and we found the root cause. I'll put it here again, in case others have the same problem:

Quirrel delivers jobs via HTTP, so it needs to know how to reach your development server.
For Blitz projects, this defaults to "http://localhost:3000".
@quatick changed their Blitz server port to 1664 - presumably because there's already something running on 3000 - so they need to tell Quirrel about the new port by setting the `QUIRREL_BASE_URL` environment variable to "http://localhost:1664".

They had though of that, but didn't include the `http://` protocol. The problem can be fixed by including it, and I've put up #216 to remove the need for it.

They mention that the UI shows scheduled/started/completed as expected - that is presumably because jobs are successfully delivered to `localhost:3000`.
